### PR TITLE
[plugin-rest-api] Add generic step to wait for JSON element presence

### PR DESCRIPTION
**Pull request by valfirst**
The step exeutes sub-steps and waits for a specified amount of time
 until HTTP response body contains an element by the specified JSON path.

### CODE REVIEW

File: vividus-util/src/main/java/org/vividus/util/function/CheckedRunnable.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.util.function;
+
+@FunctionalInterface
+public interface CheckedRunnable<E extends Exception>
```
Comment (uarlouski) [inline vividus-util/src/main/java/org/vividus/util/function/CheckedRunnable.java]: maybe Exception  -> Throwable to cover all possible cases?

Comment (valfirst) [reply vividus-util/src/main/java/org/vividus/util/function/CheckedRunnable.java]: "checked" here means checked exception-s only

Comment (uarlouski) [reply vividus-util/src/main/java/org/vividus/util/function/CheckedRunnable.java]: Throwable is also checked by its nature, if you meant that by doing this we would be able to declare also Error and RE exception then I think such cases should be prohibited by static code analysis tools

Comment (valfirst) [reply vividus-util/src/main/java/org/vividus/util/function/CheckedRunnable.java]: Please propose the rule for static code analysis :bulb:

Comment (uarlouski) [reply vividus-util/src/main/java/org/vividus/util/function/CheckedRunnable.java]: there is such rule in PMD https://github.com/pmd/pmd/issues/1692

Comment (valfirst) [reply vividus-util/src/main/java/org/vividus/util/function/CheckedRunnable.java]: this rule about method declaration, how to apply it to this case?

Comment (uarlouski) [reply vividus-util/src/main/java/org/vividus/util/function/CheckedRunnable.java]: mixed this up :)

**Pull request merged**